src/hoc/withTheme.js

```diff
@@ -42,7 +42,7 @@ const wrapWithTheme = (Component: ReactClass<any>) => {
       const themeProp = determineTheme(this.props, undefined, defaultProps)
```

Comment:
We won't need this anymore, I'd say

Reply:
Removing it causes a bug. The hoc will use the theme from defaultProp instead of the one from the themeProvider since it won't be able to tell the difference between a theme from defaultProp and a theme passed as prop